`git_pr_release_branch_staging`に渡した文字列を含むbranchがある場合、pull requestが作られない

## 事象

`git_pr_release_branch_staging`に渡した文字列を含むbranchがある場合、以下のステップで失敗してしまい、pull requestが作られない。


```
Run ! (git branch -a | grep -c "remotes/origin/$GIT_PR_RELEASE_BRANCH_STAGING")
  ! (git branch -a | grep -c "remotes/origin/$GIT_PR_RELEASE_BRANCH_STAGING")
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
    GIT_PR_RELEASE_TOKEN: ***
    GIT_PR_RELEASE_BRANCH_STAGING: release
    GIT_PR_RELEASE_BRANCH_PRODUCTION: main
    GIT_PR_RELEASE_TEMPLATE: .git-pr-release-template
1
Error: Process completed with exit code 1.
```

<img width="729" alt="image" src="https://user-images.githubusercontent.com/1811616/165219895-a5a4559c-27b6-4197-afca-76425b813654.png">


## 再現手順

以下のような状況にて当workflowを実行する。

- `git_pr_release_branch_staging`に`release`を設定
- `release` branchは存在しない
- `release_foo` branchは存在する

## 期待する挙動

リリースPRが作成される。

## 実際の挙動

pull requestが作られない。

--- a/.github/workflows/release-pr.yml
+++ b/.github/workflows/release-pr.yml
@@ -53,7 +53,7 @@ jobs:
       - name: Check if release branch exists
         if: ${{ !inputs.override }}
         run: |
-          ! (git branch -a | grep -c "remotes/origin/$GIT_PR_RELEASE_BRANCH_STAGING")
+          ! (git branch -a --format="%(refname:short)" | grep -cx "origin/$GIT_PR_RELEASE_BRANCH_STAGING")
 
       - name: Create new Branch
         run: |

